### PR TITLE
Vendor extensions: Add Nuclei vendor extensions

### PR DESCRIPTION
**Pull request by fanghuaqi**
This patch defines the Nuclei Instruction prefix `xl`, abbreviation of XinLai(Chinese pronunciation of Nuclei 芯来科技).

### CODE REVIEW

File: README.mkd [hunk 1]
```diff
@@ -299,6 +299,7 @@ Vendor                 | Prefix          | URL
 SiFive                 | sf              | https://www.sifive.com/
 T-Head                 | th              | https://www.t-head.cn/
 Ventana Micro Systems  | vt              | https://www.ventanamicro.com/
+Nuclei                 | xl              | https://nucleisys.com/
```
Comment (kito-cheng) [inline README.mkd:302]: Just want to make sure that: `xl` means Xin-Lai (芯来), right?

Comment (fanghuaqi) [reply README.mkd:302]: Yes

Comment (kito-cheng) [reply README.mkd:302]: I would like add a NOTE below the table:
e.g.

```
NOTE: The Nuclei instruction prefix `xl` is an abbreviation of "XinLai," which is the Chinese pronunciation of Nuclei.
```

(Rewrite by ChatGPT, just a suggestion :P)

Comment (fanghuaqi) [reply README.mkd:302]: Thanks for suggestion, done with new changes

**Review by kito-cheng — APPROVED** [review]
LGTM

**Comment by kito-cheng** [comment]
Just wait few more days to see if any other comment or concern.

**Review by pz9115 — APPROVED** [review]

**Review by cmuellner — APPROVED** [review]

**Pull request merged**